pkg/common/constants/constants.go

```diff
@@ -55,7 +55,7 @@ const AppManagerHandlerName = "yunikorn-app"
 const PlaceholderContainerImage = "k8s.gcr.io/pause"
 const PlaceholderContainerName = "pause"
 const PlaceholderPodRestartPolicy = "Never"
-const AnnotationPlaceholderFlag = "yunikorn.apache.org/placeholder"
+const PlaceholderFlag = "yunikorn.apache.org/placeholder"
```

Comment:
For the `ApplicationID` we use an unqualified `applicationId` in labels and a qualified version `yunikorn.apache.org/app-id` in the annotation. Shouldn't we follow the same rule here? I can live with either solution, I just think we need to set a standard for what we do.

Reply:
Sure, follow the some rule will more clear to read.